components/myforms/HTMLDownload/Table.tsx

```diff
@@ -274,9 +272,7 @@ export const Table = (props: TableProps): React.ReactElement => {
                 {t("responseTemplate.submissionDate", { lng: lang })}
               </dt>
               <dd className="py-4">
-                {new Date(submissionDate).toLocaleString(`${lang + "-CA"}`, {
-                  timeZone: Intl.DateTimeFormat().resolvedOptions().timeZone,
-                })}
+                {new Date(submissionDate).toISOString().replace("T", " ").slice(0, -5) + " UTC"}
```

Comment:
Since both dates (EN or FR) have the same formatting we could probably refactor it (using a `const`) so that we only have to change the code in one specific place if we need to